agent/src/main/java/org/openjdk/jmc/agent/jmx/AgentControllerMXBean.java

```diff
@@ -38,5 +38,12 @@
 
 	public void defineEventProbes(String xmlDescription) throws Exception;
```

Comment:
Javadoc here explaining when exceptions are thrown (I assume when providing an invalid probe definition).

Reply:
Javadoc added in 80b5757